src/main/java/training/chessington/model/pieces/Rook.java

```diff
@@ -15,6 +15,36 @@ public Rook(PlayerColour colour) {
 
     @Override
     public List<Move> getAllowedMoves(Coordinates from, Board board) {
-        return new ArrayList<>();
+
+        List<Move> allowableMoves = new ArrayList<>();
+        addEmptySpaces(from, board, allowableMoves, 1, 0);   // down
+        addEmptySpaces(from, board, allowableMoves, -1, 0);   // up
+        addEmptySpaces(from, board, allowableMoves, 0, 1);   // right
+        addEmptySpaces(from, board, allowableMoves, 0, -1);   // left
+
+
+        return allowableMoves;
+    }
+
+    private void addEmptySpaces(Coordinates from, Board board, List<Move> initAllowableMoves, int vertical, int horizontal) {

```

Comment:
This method is really good. I think the only thing I'd change is the name as it doesn't just add the empty spaces, it also adds a square with an enemy piece in it. I think maybe something like `addAllowedMovesInOneDirection` (or if you can think of something better) :D